src/components/StyledButton.js

```diff
@@ -1,6 +1,6 @@
 import styled from "@emotion/styled"
 
-const CalltoActionButton = styled.button`
+const StyledButton = styled.button`
```

Comment:
Any reason this is called `StyledButton`? 
I would just call it `Button`. The name should give a hint of the functionality or use case of a component, like in normal HTML.

If you have a button and style it, you wouldn't call the CSS class `styled` like `<button class="styled">`.
The name `styled` doesn't give you useful information. Instead, you would style all buttons or create a class `.button` like `<button class="button">`.

If you have a specifict use case, the class gives you more information like `<button class="danger">`. This would result in a styled component called `DangerButton`.